.github/workflows/release-30_publish_release_draft.yml

```diff
@@ -0,0 +1,161 @@
+          CORETIME_WESTEND_DIGEST: ${{ github.workspace}}/coretime-westend-runtime/coretime-westend-srtool-digest.json
+          GLUTTON_WESTEND_DIGEST: ${{ github.workspace}}/glutton-westend-runtime/glutton-westend-srtool-digest.json
+          PEOPLE_ROCOCO_DIGEST: ${{ github.workspace}}/people-rococo-runtime/people-rococo-srtool-digest.json
+          PEOPLE_WESTEND_DIGEST: ${{ github.workspace}}/people-westend-runtime/people-westend-srtool-digest.json
+          ROCOCO_DIGEST: ${{ github.workspace}}/rococo-runtime/rococo-srtool-digest.json
+          WESTEND_DIGEST: ${{ github.workspace}}/westend-runtime/westend-srtool-digest.json
+        run: |
+          . ./.github/scripts/common/lib.sh
+
+          export REF1=$(get_latest_release_tag)
+          if [[ -z "${{ inputs.version }}" ]]; then
+            export REF2="${{ github.ref }}"
+          else
+            export REF2="${{ inputs.version }}"
+          fi
+          echo "REL_TAG=$REF2" >> $GITHUB_ENV
+          export VERSION=$(echo "$REF2" | sed -E 's/^v([0-9]+\.[0-9]+\.[0-9]+).*$/\1/')
+
+          ./scripts/release/build-changelogs.sh
+
+          echo "Checking the folder state"
+          pwd
+          ls -la scripts/release
+
+      - name: Archive artifact context.json
+        uses: actions/upload-artifact@5d5d22a31266ced268874388b861e4b58bb5c2f3 # v4.3.1
+        with:
+          name: release-notes-context
+          path: |
+            scripts/release/context.json
+            **/*-srtool-digest.json
+
+      - name: Create draft release
+        id: create-release
+        uses: actions/create-release@0cb9c9b65d5d1901c1f53e5e66eaf4afd303e70e # v1.1.4
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ env.REL_TAG }}
+          release_name: Polkadot ${{ env.REL_TAG }}
+          body_path: ${{ github.workspace}}/scripts/release/RELEASE_DRAFT.md
+          draft: true
+
+  publish-runtimes:
+    needs: [ build-runtimes, publish-release-draft ]
+    continue-on-error: true
+    runs-on: ubuntu-latest
+    strategy:
+      matrix: ${{ fromJSON(needs.build-runtimes.outputs.published_runtimes) }}
+
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4.0.0
+
+      - name: Download artifacts
+        uses: actions/download-artifact@c850b930e6ba138125429b7e5c93fc707a7f8427 # v4.1.4
+
+      - name: Get runtime info
+        env:
+          JSON: release-notes-context/${{ matrix.chain }}-runtime/${{ matrix.chain }}-srtool-digest.json
+        run: |
+          >>$GITHUB_ENV echo ASSET=$(find ${{ matrix.chain }}-runtime -name '*.compact.compressed.wasm')
+          >>$GITHUB_ENV echo SPEC=$(<${JSON} jq -r .runtimes.compact.subwasm.core_version.specVersion)
+
+      - name: Upload compressed ${{ matrix.chain }} v${{ env.SPEC }} wasm
+        if: ${{ matrix.chain }} !~= 'rococo-parachain'
```

Comment:
What is this doing?

Reply:
It is here to exclude the `rococ-parachain` from publishing in the draft. Was made with typos here, already corrected it.